config/config.default.yaml

```diff
@@ -591,6 +590,19 @@ sector:
     # - onshore  # more than 50 km from sea
   - nearshore    # within 50 km of sea
     # - offshore
+  methanol: false
```

Comment:
it is maybe better to have all of the technologies as a nested dictionary, if you want to have this switch
```
methanol:
    enable: true
    regional_methanol_demand: false
    [...]
```

Reply:
changes it to your suggestions.